### PR TITLE
cockroach 1.0.1

### PR DESCRIPTION
**Pull request by benesch**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/cockroach.rb [hunk 1]
```diff
@@ -38,9 +38,9 @@ def caveats; <<-EOS.undent
   plist_options :manual => "cockroach start --insecure"
 
   def plist; <<-EOS.undent
-    <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-    <plist version="1.0">
+    <?xml version="1.0.1" encoding="UTF-8"?>
```
Comment (mberhault) [inline Formula/cockroach.rb]: uh. that's not right.

Comment (benesch) [inline Formula/cockroach.rb]: Ha! Looks like a bug in `brew bump-formula-pr`. Welp. Looks like it'll only happen this once.

**Review by benesch — COMMENTED** [review]
Ok, fixed. Nice catch, @mberhault!

File: Formula/cockroach.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Cockroach < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  url "https://binaries.cockroachdb.com/cockroach-v1.0.src.tgz"
-  version "1.0"
-  sha256 "ca87b10eec688195e0df4f85431b019f2980ae4b511ee321f91f945315efeb76"
+  url "https://binaries.cockroachdb.com/cockroach-v1.0.1.1.src.tgz"
```
Comment (mberhault) [inline Formula/cockroach.rb]: this is off too, extra `.1`

Comment (benesch) [inline Formula/cockroach.rb]: 🤦‍♂️

**Comment by mberhault** [comment]
LGTM

**Comment by dunn** [comment]
Thank you both! Merged as e3d3a6d6a05fa87357ac7b4f8506105046a3d0fd.

**Pull request closed without merging**